src/js/elements/mapchip/legend.js

```diff
@@ -32,18 +32,17 @@ export class Legend extends Component {
         this._clonedLegend = $('.map-options__legend')[0].cloneNode(true);
     }
 
-    show(colors, intervalLabels, lightStart = 3) {
-        if (colors.length != intervalLabels.length)
+    show(colors, intervalLabels) {
+        if (colors.length !== intervalLabels.length)
             throw 'Expected the number of intervals to be the same as the number of colours.'
 
         this.isLoading = false;
 
         for (let i = 0; i < intervalLabels.length; i++) {
             const interval = intervalLabels[i];
             const item = this._clonedLegendBlock.cloneNode(true);
-            const label = interval;
 
-            if (i >= lightStart) {
+            if (!isColorLight(colors[i])) {
```

Comment:
I like that the comment explaining isColorLight is in the utils file, so it doesn't clutter this code but is easy to find if someone looks deeper